ImagePicker/CellRegistrator.swift

```diff
@@ -160,19 +114,32 @@ public final class CellRegistrator {
         assetItemClassesData?[type] = (cellClass, cellIdentifier)
     }
     
-    ///
     /// Register a cell class for all asset items types (image and video).
-    ///
     public func registerCellClassForAssetItems<T: UICollectionViewCell>(_ cellClass: T.Type) where T: ImagePickerAssetCell {
         assetItemClass = cellClass
     }
     
-    ///
     /// Register a cell nib for all asset items types (image and video).
     ///
     /// Please note that cell's class must conform to `ImagePickerAssetCell` protocol, otherwise an exception will be thrown.
-    ///
     public func registerNibForAssetItems(_ nib: UINib) {
         assetItemNib = nib
     }
+    
+    func cellIdentifier(forActionItemAt index: Int) -> String? {
+        // First lets check if there is a registered cell at specified index
+        if let index = actionItemNibsData?[index]?.1 ?? actionItemClassesData?[index]?.1 {
```

Comment:
put under guard condition

Reply:
I think its inteded to be as it is. Its some obscure recursion